<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resultado</class>
 <widget class="QWidget" name="Resultado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FLOUWER</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>999</width>
     <height>999</height>
    </size>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="contadorLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>670</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="botonVolver_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string> Volver al menu</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/Menu.png</normaloff>:/images/images/Menu.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="diasFertiles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>419</width>
     <height>99</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
